networking/hardware_networks/configuring-sriov-device.adoc

```diff
@@ -16,8 +16,10 @@ include::modules/nw-sriov-configuring-device.adoc[leveloffset=+1]
 
 include::modules/nw-sriov-troubleshooting.adoc[leveloffset=+1]
 
-:FeatureName: CNI VRF plug-in
-include::modules/technology-preview.adoc[leveloffset=+1]
+[NOTE]
+====
+Using a VRF through the `ip vrf exec` command is not supported in {product-title} pods. To use VRF, applications can bind directly to the VRF interface.
+====
```

Comment:
should this go in the module below `modules/cnf-assigning-a-sriov-network-to-a-vrf.adoc` together with the other note?